v4/test/graphql/graphql_test_helper.go

```diff
@@ -0,0 +1,53 @@
+package graphql
```

Comment:
shouldn't the file be named `graphql_helper_test.go`? are we planning to use this file outside tests?

Reply:
fixed